vere/dawn.c

```diff
@@ -164,6 +164,24 @@ _dawn_fail(u3_noun who, u3_noun rac, u3_noun sas)
   u3_lo_bail();
 }
 
+/* _dawn_need_unit(): produce a value or pail
```

Comment:
pail ... you mean bail?

Reply:
Also, this feels like it should be generic Vere functionality.

Reply:
whoops! I could call `+need`, but that's an %exit bail, which dumps core on the home road. I need to call `u3_lo_bail()`, which exits a little more cleanly. And i need to print an error message first.

Reply:
Oh, ok. I think you should add a comment explaining that. Otherwise a year from now some poor soul will try to move it and end up in a world of hurt.